src/test/java/de/retest/recheck/review/ignore/PixelDiffFilterTest.java

```diff
@@ -96,4 +96,14 @@ void style_attribute_should_return_false() {
 		assertThat( cut.matches( mock( Element.class ), diff ) ).isFalse();
 	}
 
+	@Test
+	void boxShadow_should_not_be_parsed() throws Exception {
+		final String expected =
+				"rgba(0, 0, 0, 0.03) 0px 0px 1.18247px 0px, rgba(0, 0, 0, 0.06) 0px 1.18247px 2.36495px 0px";
+		final String actual =
+				"rgba(0, 0, 0, 0.098) 0px 0px 3.90236px 0px, rgba(0, 0, 0, 0.196) 0px 3.90236px 7.80472px 0px";
+		final AttributeDifference difference = new AttributeDifference( "box-shadow", expected, actual );
+
+		assertThat( cut.matches( mock( Element.class ), difference ) ).isFalse();
```

Comment:
I know this was there before, but do you think it would make sense to extract the mocked `Element` into a field?